http/servlet-undertow/pom.xml

```diff
@@ -23,5 +23,19 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-security</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx</artifactId>
```

Comment:
I suspect this dependency is only serving for your tests so it can be in the test scope?

Reply:
yes put it in test scope